meta/src/main/proto/gravitino_meta.proto

```diff
@@ -114,4 +114,14 @@ message Group {
   string name = 2;
   repeated string roles = 3;
   AuditInfo audit_info = 4;
-}
\ No newline at end of file
+}
+
+message Role {
+  uint64 id = 1;
+  string name = 2;
+  repeated string privileges = 3;
+  string privilege_entity_identifier = 4;
+  string privilege_entity_type = 5;
+  map<string, string> properties = 6;
```

Comment:
For table, we use properties to distinguish which table is created by Gravitno. I add properties for the similar purpose.